deploy/helm/templates/deployment.yaml

```diff
@@ -79,6 +79,8 @@ spec:
               value: "{{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.tools.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
             - name: KUBEBLOCKS_CHARTS_IMAGE
               value: "{{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.charts.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+            - name: KUBEBLOCKS_DATASCRIPT_CLIENTS_IMAGE
+              value: "{{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.datascript.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
also place this image in init. containers to assure presence of the image tag

Reply:
Got it.  updated.